lib/domain/entities/activity.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:movna/domain/entities/sport.dart';
+import 'package:movna/domain/entities/track_point.dart';
+import 'package:movna/domain/entities/track_segment.dart';
+
+part 'activity.freezed.dart';
+
+/// Training activity session.
+@freezed
+class Activity with _$Activity {
+  const Activity._();
+
+  const factory Activity({
+    /// The time at which this activity begun.
+    required DateTime startTime,
+
+    /// The time at which this activity ended.
+    /// Null when activity is not ended yet.
+    DateTime? stopTime,
+
+    /// The sport practiced during this activity. Can be unspecified.
+    Sport? sport,
+
+    /// The name of this activity. Can be unspecified.
+    String? name,
+
+    /// The total distance traveled during this activity in meters.
+    @Default(0) double distanceInMeters,
+
+    /// The total duration of this activity, accounting for pauses.
```

Comment:
I'm confused by this, if I start the activity at 15:00, end it at 16:00 and make a 15min pause in the middle, will the duration be 45min et 1h?
I think it'd be clearer using `includingPauses` if the result should be 1h or `excludingPauses` otherwise.

The question remaining is, why isn't the other duration available here ?

Reply:
Yes I see that my comment is unclear, when I say accounting for pauses, I mean that the pauses duration is subtracted from the total time. The `durationIncludingPauses` can be recovered using `stopTime-startTime`